package.json

```
@@ -51,6 +51,13 @@
   "dependencies": {
     "@hono/node-server": "1.8.2",
     "@notionhq/client": "2.2.14",
+    "@opentelemetry/api": "^1.8.0",
+    "@opentelemetry/exporter-prometheus": "^0.49.1",
+    "@opentelemetry/exporter-trace-otlp-http": "^0.49.1",
+    "@opentelemetry/resources": "^1.22.0",
+    "@opentelemetry/sdk-metrics": "^1.22.0",
+    "@opentelemetry/sdk-trace-base": "^1.22.0",
+    "@opentelemetry/semantic-conventions": "^1.22.0",
```

```suggestion
    "@opentelemetry/api": "1.8.0",
    "@opentelemetry/exporter-prometheus": "0.49.1",
    "@opentelemetry/exporter-trace-otlp-http": "0.49.1",
    "@opentelemetry/resources": "1.22.0",
    "@opentelemetry/sdk-metrics": "1.22.0",
    "@opentelemetry/sdk-trace-base": "1.22.0",
    "@opentelemetry/semantic-conventions": "1.22.0",
```
Please pin the dependency version.